### ISSUE TITLE
Broken HTML in base template file

### ISSUE THREAD

**Issue by JoshuaSoileau** [magento/magento2#6081]
The HTML in the following file is broken:

```
app/code/Magento/Review/view/frontend/templates/form.phtml
line #25
```

Here is the line with broken HTML:

```
<legend class="label"><span><?php /* @escapeNotVerified */ echo __('Your Rating') ?><span></legend><br/>
```

[Here](https://github.com/magento/magento2/blob/develop/app/code/Magento/Review/view/frontend/templates/form.phtml) is a link to said file

The second `<span>` tag should be a closing tag (`</span>`)
### Preconditions

1. Google Chrome web browser
### Steps to reproduce

1. View product page on frontend
2. Inspect "Your Rating" text on review submission form
3. View DOM markup for `legend` element
### Expected result

1. HTML markup in DOM should only contain 1 `span` tag
### Actual result

1. HTML markup contains 2 `span` tags, as the browser will course correct and close the tag itself.
   ![image](https://cloud.githubusercontent.com/assets/2104388/17526548/b11ddba2-5e35-11e6-94f1-789417f1f508.png)

**Comment by AVoskoboinikov** [comment]
Hello,

Thank you for reporting this issue. I have created ticket for it in our internal Jira (MAGETWO-57428). This issue will be closed after fix.

**Comment by Ctucker9233** [comment]
I believe there is already a pull request that fixes this issue.

**Comment by Igloczek** [comment]
It's already fixed on `develop` branch - https://github.com/magento/magento2/commit/5228cf78543e27de4aebe79d003efc2e0af35922

**Comment by ishakhsuvarov** [comment]
@Igloczek Thanks for backporting!

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-75319

**Comment by magento-engcom-team** [comment]
Fixed in 2.2.0 and 2.1
https://github.com/magento/magento2/blob/2.1-develop/app/code/Magento/Review/view/frontend/templates/form.phtml#L25
Thank you for report
